Integrations/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM deephaven/java-and-python:local-build as install-wheels
+
+RUN set -eux; \
+    apt-get -qq update; \
+    apt-get -qq -y --no-install-recommends install liblzo2-2 curl; \
+    rm -rf /var/lib/apt/lists/*
+
+COPY deephaven-jpy-wheel/ /deephaven-jpy-wheel
+RUN set -eux; \
+    python3 -m pip install -q --no-cache-dir /deephaven-jpy-wheel/*.whl; \
+    rm -r /deephaven-jpy-wheel
+
+COPY deephaven-wheel/ /deephaven-wheel
+RUN set -eux; \
+    python3 -m pip install -q --no-cache-dir /deephaven-wheel/*.whl; \
+    rm -r /deephaven-wheel
+
+COPY deephaven2-wheel/ /deephaven2-wheel
+RUN set -eux; \
+    python3 -m pip install -q --no-cache-dir /deephaven2-wheel/*.whl; \
+    rm -r /deephaven2-wheel
+
+FROM deephaven/java-and-python:local-build
+COPY --from=install-wheels / /
```

Comment:
Do I understand properly that these last two lines "start over" with the fresh image, and just put one big layer on top, with the contents of the above steps? 

Assuming so, would it be possible to just mash in the created dist-packages from the three above RUNs, and let each RUN happen in its own distinct FROM, so that they don't affect each other? This is probably overkill, but since we're making an entirely fresh 200mb build each time the above 200mb build changes, it seems like we could have some room to improve a bit, or drop this "flattening" step and cut our churn in half?

Reply:
Yes - we are taking the changes from the "install-wheels" stage, and just adding the bits that have changed back against java-and-python:local-build.

The reason I don't want to run 3 pip installs runs to be parallel (instead of serial) is that I can't guarantee they won't have overlapping bits; and I'd prefer to let the pip package manager deal with any potential version conflicts than either breaking or silently choosing one during a copy stage.